src/protocols/LoRaWAN/LoRaWANBands.cpp

```diff
@@ -20,16 +20,18 @@ const LoRaWANBand_t* LoRaWANBands[RADIOLIB_LORAWAN_NUM_SUPPORTED_BANDS] = {
 const LoRaWANBand_t EU868 = {
   .bandNum = BandEU868,
   .bandType = RADIOLIB_LORAWAN_BAND_DYNAMIC,
-  .payloadLenMax = {  59,  59,  59, 123, 230, 230, 230, 230,   0,   0,   0,   0,   0,   0,   0 },
+  .payloadLenMax = {  59,  59,  59, 123, 250, 250, 250, 250,  58, 123,  58, 123,   0,   0,   0 },
+  .payloadLenRel = {  59,  59,  59, 123, 230, 230, 230, 230,  58, 123,  58, 123,   0,   0,   0 },
```

Comment:
Perhaps I'm missing some context (haven't gone through the bulk of changes yet, but it looks to me that `payloadLenRel` could be replaced by something like `max(payloadLenMax[i], 230)`.

Reply:
That was something I found out only after adding all of those values... Honestly quite puzzled me as the faster data rates are those that do _not_ hit dwell time limits or very long ToA in general, but they are allowed additional metadata.. oh well. Will do.